client/lib/signup/site-type.js

```diff
@@ -27,6 +27,55 @@ export function getSiteTypePropertyValue( key, value, property, siteTypes = getA
 	return get( siteTypeProperties, property, null );
 }
 
+const copyDefaults = {
+	// General copy
+	label: i18n.translate( 'Default label' ),
+	description: i18n.translate( 'default description' ),
+	previewScrollDownCopy: i18n.translate(
+		'Scroll down to see your site. Once you complete setup you’ll be able to customize it further.'
+	),
+	// Site title step
+	siteTitleLabel: i18n.translate( 'Give your site a name' ),
+	siteTitleSubheader: i18n.translate(
+		'This will appear at the top of your site and can be changed at anytime.'
+	),
+	siteTitlePlaceholder: i18n.translate( 'default siteTitlePlaceholder' ),
+	// Site topic step
+	siteTopicHeader: i18n.translate( 'What is your site about?' ),
+	siteTopicSubheader: i18n.translate(
+		"We'll add relevant content to your site to help you get started."
+	),
+	siteTopicInputPlaceholder: i18n.translate( 'Enter a topic or choose one from below.' ),
+	// Domains step
+	domainsStepHeader: i18n.translate( 'Give your site an address' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Give your site an address.' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).